diff --git a/CMakeLists.txt b/CMakeLists.txt
index eef6626..6d724ba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -102,6 +102,7 @@ include(AbseilHelpers)
 ##
 ## DO NOT rely on the internal targets outside of the prefix
 
+add_compile_definitions(ABSL_FORCE_WAITER_MODE=4)
 
 # include current path
 list(APPEND ABSL_COMMON_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR})
@@ -181,7 +182,7 @@ endif()
 add_subdirectory(absl)
 
 if(ABSL_ENABLE_INSTALL)
-  
+
 
   # install as a subdirectory only
   install(EXPORT ${PROJECT_NAME}Targets
